packages/esm-appointments-app/src/appointments/common-components/appointments-actions.test.tsx

```diff
@@ -48,6 +48,18 @@ const appointment: Appointment = {
   extensions: [],
 };
 
+let visits;
```

Comment:
Is using a let like this a best pratice for defining a mocked function when I want to change the return value between tests?

Reply:
You lose TypeScript's implicit typing this way. `visits` now has an `any` type. I'd just define them locally in the `it()` functions.

Reply:
I needed to find away to change the return value between test runs, I think I figured out the best practice, updating the PR.